components/grid/columns/resize.md

```diff
@@ -25,6 +25,8 @@ To enable such column resizing, set the `Resizable` parameter of the grid to `tr
 
 To prevent the user from resizing a certain column, set its own parameter `Resizable="false"`. Note that the user can still resize other columns around it.
 
+If the column `Width` is less than `MinResizableWidth` and the user tries to resize the column, it will snap to its minimum width. When using [multi-column headers]({%slug grid-columns-multiple-column-headers%}) and there is a conflict between the `MinResizableWidth` configuration of the parent and child column, then the child column setting will take precedence. The same logic applies for `MaxResizableWidth`.
```

Comment:
should add a sentence like the first one but for max size being lower than current size.